### DISCUSSION TITLE
Future Improvement Plans & Goals

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [webdriverio/webdriverio#8216] [General]
Hello!
Where I can find some information about the future improvements and goals that are planned for WebDriverIO updates?!

**Comment by christian-bromann** [comment]
Please take a look at our [roadmap](https://github.com/webdriverio/webdriverio/blob/main/ROADMAP.md). This is not an exhaustive list and up for change whenever someone suggests something that makes sense for all. Feel free to get involved in any of the items or suggest new things 😉. Happy to help onboarding anyone for contributions.
